vsphere/internal/helper/hostsystem/host_system_helper.go

```diff
@@ -51,7 +51,7 @@ func FromID(client *govmomi.Client, id string) (*object.HostSystem, error) {
 	defer cancel()
 	hs, err := finder.ObjectReference(ctx, ref)
 	if err != nil {
-		return nil, fmt.Errorf("could not find host system with id: %s: %s", id, err)
+		return nil, err
```

Comment:
Because the error messages coming from the API can be a bit vague sometimes. Adding a little bit of text gives better context and we can also track the source of the error just by grepping for the error message if we have to :) .

Reply:
You went the other way here though, right? It looks like it originally included "could not find host system with..." and that part was removed.

Reply:
Hmm you're right.... I guess I read the diff the other way. This must have happened by accident. The good news is that I have another branch I'm working on where I'm revisiting all this code and adding some text instead of just returning the vsphere error, so this will be adressed there.